docs/running-on-kubernetes.md

```diff
@@ -111,70 +162,96 @@ If our local proxy were listening on port 8001, we would have our submission loo
       --conf spark.app.name=spark-pi \
       --conf spark.kubernetes.driver.docker.image=kubespark/spark-driver:v2.1.0-kubernetes-0.1.0-alpha.2 \
       --conf spark.kubernetes.executor.docker.image=kubespark/spark-executor:v2.1.0-kubernetes-0.1.0-alpha.2 \
+      --conf spark.kubernetes.initcontainer.docker.image=kubespark/spark-init:v2.1.0-kubernetes-0.1.0-alpha.2 \
```

Comment:
I like that idea -- let's discuss separately: https://github.com/apache-spark-on-k8s/spark/issues/289